protobuf/plugin/fieldpath/helpers.go

```diff
@@ -14,14 +14,15 @@
    limitations under the License.
 */
 
-package plugin
+package fieldpath
 
 import (
+	"github.com/containerd/containerd/protobuf/plugin"
 	"github.com/gogo/protobuf/proto"
 	"github.com/gogo/protobuf/protoc-gen-gogo/descriptor"
 )
 
 // FieldpathEnabled returns true if E_Fieldpath is enabled
-func FieldpathEnabled(file *descriptor.FileDescriptorProto, message *descriptor.DescriptorProto) bool {
-	return proto.GetBoolExtension(message.Options, E_Fieldpath, proto.GetBoolExtension(file.Options, E_FieldpathAll, false))
+func FieldpathEnabled(file *descriptor.FileDescriptorProto, message *descriptor.DescriptorProto) bool { //nolint: golint // Ignoring failure - "func name will be used as fieldpath.FieldpathEnabled by other packages, and that stutters; consider calling this Enabled"
```

Comment:
Since we are breaking this api anyways, I think its fine to call this `Enabled` to satisfy the style here.

Reply:
Done!